modules/security-context-constraints-about.adoc

```diff
@@ -132,6 +152,16 @@ The `restricted` SCC:
 The restricted SCC is the most restrictive of the SCCs that ship by default with the system. However, you can create a custom SCC that is even more restrictive. For example, you can create an SCC that restricts `readOnlyRootFS` to `true` and `allowPrivilegeEscalation` to `false`.
 ====
 
+|`restricted-v2`
+| Like the `restricted` SCC, but with the following differences:
+
+* `ALL` capabilities are dropped from containers.
+* The `NET_BIND_SERVICE` capability can be added explicitly.
```

Comment:
The text gives the impression that we can use `NET_BIND_SERVICE ` and be qualified for restricted-v2.
Therefore for restricted-v2, we need to drop ALL or leave empty. 

c/ @stlaz